### DISCUSSION TITLE
twitter?

### DISCUSSION THREAD

**Discussion by aoandres** [sherlock-project/sherlock#906] [Q&A]
was Twitter removed? it was mayor contributor on finding people by username...?
also, https://ok.ru/ has a massive database of people just like www.vk.com ...to find Russians.( just a suggestion)

**Comment by jcs090218** [comment]
It seems like Twitter isn't removed from Sherlock. See document [removed_sites.md](https://github.com/sherlock-project/sherlock/blob/master/removed_sites.md) for more information.

**Comment by franklinjosh11** [comment]
It was a good platform to find out people related to your community by their usernames then I seen the reason behind this. It [seems](https://pingpongbuzz.com/) they didn't removed.

**Comment by jcs090218** [comment]
Update, I think someone has reported `false-negative` on Twitter. See #913.

**Comment by SethFalco** [comment]
Twitter was removed, however it was brought back in https://github.com/sherlock-project/sherlock/commit/65e1d2c82b015793e30de330b44fa2ffd189eefe on 25th October 2021.

**Reply by Cfomodz** [reply]
Answered and can be marked as such :)
